cluster/gce/upgrade.sh

```diff
@@ -213,6 +251,11 @@ fi
 
 prepare-upgrade
 
+if [[ "${node_prereqs}" == "true" ]]; then
+  prepare-node-upgrade
+  exit 0
```

Comment:
Should the exit code depend on whether prepare-node-upgrade is successful (or maybe that is implicit)?